frontends/mit-open/src/pages/DashboardPage/SettingsPage.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react"
+import {
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+  Link,
+  styled,
+} from "ol-components"
+import { useUserMe } from "api/hooks/user"
+import {
+  useSearchSubscriptionDelete,
+  useSearchSubscriptionList,
+} from "api/hooks/searchSubscription"
+
+const SOURCE_LABEL_DISPLAY = {
+  topic: "Topic",
+  unit: "MIT Unit",
+  department: "MIT Academic Department",
+  "saved search": "Saved Search",
+}
+
+const FollowList = styled(List)(() => ({
+  borderRadius: "4px",
+  background: "#fff",
+  padding: "0px",
+  margin: "0px",
+}))
+
+const StyledLink = styled(Link)(({ theme }) => ({
+  color: theme.custom.colors.red,
+}))
+
+const TitleText = styled(Typography)(({ theme }) => ({
+  marginTop: "20px",
+  marginBottom: "5px",
+
+  color: theme.custom.colors.darkGray2,
+  ...theme.typography.subtitle1,
+}))
+
+const SubTitleText = styled(Typography)(({ theme }) => ({
+  marginBottom: "10px",
```

Comment:
Curious: Did 10px come from Figma? Steve and Bilal usually use multiples of 8 essentially always use at least multiples of 4.

**Request**: Could you double check the margins/paddings/gap.

Reply:
checked and fixed